flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -977,6 +977,8 @@ static const IntrinsicInterface genericIntrinsicFunction[]{
         DefaultInt, Rank::vector, IntrinsicClass::transformationalFunction},
     {"this_image", {OptionalTEAM}, DefaultInt, Rank::scalar,
         IntrinsicClass::transformationalFunction},
+    {"time", {}, TypePattern{IntType, KindCode::exactKind, 8}, Rank::scalar,
+        IntrinsicClass::transformationalFunction},
```

Comment:
`IntrinsicClass::transformationalFunction` is it the correct class?

Reply:
I am open to other suggestions, what would you expect it to be?
It wasn't the obvious choice to me, but it does seem to fit according to how the standard classifies things.

- I would have guessed it was an inquiry function, but those are defined as functions whose values depend on the properties of it's arguments as opposed to there values. 
- It doesn't seem like it should be elemental since it doesn't take an argument at all.
- Transformational functions are defined as being intrinsics that are neither elemental or inquiry functions.

Reply:
I think you can just omit it and it will have no class.

Reply:
Maybe noClass, but that doesn't seem to exist by the standards definitions and didn't have any documentation.

Reply:
`IntrinsicClass::transformationalFunction` is fine if there is no other class that fits.

Reply:
This table is elemental by default.

Reply:
The only use of noClass that I could find was as the default return value of `GetIntrinsicClass`.